Core/Src/CAN/general_can.c

```diff
@@ -59,3 +62,12 @@ void CAN_general_receive_callback(FDCAN_HandleTypeDef* hfdcan, uint32_t RxFifo0I
 		}
 	}
 }
+
+
+
+void CAN_general_error_callback(FDCAN_HandleTypeDef *hfdcan, uint32_t ErrorStatusITs){
+    if(ErrorStatusITs == FDCAN_IT_BUS_OFF){
```

Comment:
```suggestion
    if(ErrorStatusITs |= FDCAN_IT_BUS_OFF){
```
porque se tiver qualquer outro erro junto com o `BUS_OFF` ele n vai cair aq